<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectListWindowBaseClass</class>
 <widget class="QMainWindow" name="ObjectListWindowBaseClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="SkTableView" name="table">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>&amp;Schließen</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Datenbank">
    <property name="title">
     <string>&amp;Datenbank</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionDelete"/>
    <addaction name="actionEdit"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menu_Datenbank"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;Neu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Löschen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>&amp;Editieren</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Aktualisie&amp;ren</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Schließen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SkTableView</class>
   <extends>QTableView</extends>
   <header>src/gui/widgets/SkTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ObjectListWindowBaseClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>744</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
